Gemfile

```diff
@@ -1,4 +1,8 @@
 source 'https://rubygems.org'
 
 gem "jekyll"
-gem "just-the-docs"
\ No newline at end of file
+gem "just-the-docs"
+gem "jekyll-seo-tags"
```

Comment:
Could you add a short explanation of what each gem does?

Reply:
Thanks for review, it's done.